docs/user-guide/basic-usage.md

```diff
@@ -76,7 +76,15 @@ about the port numbers it is listening on.
 
 This tells you that the application has successfully started, and that proxy server is listening ports 8080 for HTTP, 
 8443 for HTTPS, and that admin interface has started on port 9000 (HTTPS).
+ 
     
+# Liveness check
+
+It is possible to check when Styx is ready for serve requests by accessing an admin endpoint at `/admin/proxystatus`.
```

Comment:
As discussed, let's call this `/admin/styx/proxy/status`.

Reply:
As discussed, let's return  a JSON object for this status:
{ 
status: "UP|DOWN"
}